public/js/validations.js

```diff
@@ -1,55 +1,83 @@
 function isDate(date) {
-    let pattern = /([12]\d{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])/
+    let pattern = /([12]\d{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])/;
     if (pattern.test(date)) {
-        return true
-    } else {
-        return false
+        return true;
     }
+
+    return false;
 }
 
 function isTime(time) {
-    let pattern = /([01][0-9]|2[0-3]):[0-5][0-9]/
+    let pattern = /([01][0-9]|2[0-3]):[0-5][0-9]/;
     if (pattern.test(time)) {
-        return true
-    } else {
-        return false
+        return true;
     }
+
+    return false;
+}
+
+function isName(name) {
+    let pattern = /^[A-Za-z][A-Za-z\'\-]+([\ A-Za-z][A-Za-z\'\-]+)*/;
+    if (pattern.test(name)) {
+        return true;
+    }
+
+    return false;
 }
 
 function isPhoneNumber(phone) {
-    let regEx = /^(([+][(]?[0-9]{1,3}[)]?)|([(]?[0-9]{4}[)]?))\s*[)]?[-\s\.]?[(]?[0-9]{1,3}[)]?([-\s\.]?[0-9]{3})([-\s\.]?[0-9]{3,4})$/gm
+    let regEx = /^(([+][(]?[0-9]{1,3}[)]?)|([(]?[0-9]{4}[)]?))\s*[)]?[-\s\.]?[(]?[0-9]{1,3}[)]?([-\s\.]?[0-9]{3})([-\s\.]?[0-9]{3,4})$/gm;
     if (regEx.test(phone)) {
-        return true
-    } else {
-        return false
+        return true;
     }
+
+    return false;
```

Comment:
It appears `pattern.test()` returns a `Boolean`, so we can just return the result.